cg/meta/orders/case_submitter.py

```diff
@@ -96,23 +96,15 @@ def _validate_case_names_are_unique(
     def submit_order(self, order: OrderIn) -> dict:
         """Submit a batch of samples for sequencing and analysis."""
         result = self._process_case_samples(order=order)
-        for case_obj in result["records"]:
```

Comment:
Can we remove this?

Reply:
Sure! @islean @Vince-janv Any opinions?

Reply:
If it is only there for the LOG.info I am all for removing it!